tests/conftest.py

```diff
@@ -534,6 +534,41 @@ def sre_config_yaml(request):
     return yaml.dump(yaml.safe_load(content))
 
 
+@fixture
+def sre_config_yaml_missing_field(request):
+    content = """---
+    azure:
+        location: uksouth
+        subscription_id: guid_subscription
+        tenant_id: guid_tenant
+    description: Sandbox Project
+    dockerhub:
+        access_token: dummytoken
+        username: exampleuser
+    name: sandbox
+    sre:
+        admin_ip_addresses:
+        - 1.2.3.4/32
+        data_provider_ip_addresses: []
+        databases: []
+        remote_desktop:
+            allow_copy: false
+            allow_paste: false
+        research_user_ip_addresses: []
+        software_packages: none
+        storage_quota_gb:
+            home: 100
+            shared: 100
+        timezone: Europe/London
+        workspace_skus: []
```

Comment:
Is there already a fixture with this config (or one that is close enough)? You could use that here instead of duplicating:

```python
@fixture
def sre_config_yaml_missing_field(request, config_yaml):
    config_yaml.replace(
        ...
```